### PR TITLE
qsv: init at 0.112.0

### PR DESCRIPTION
**Pull request by cpcloud**
###### Description of changes

Initialize the [`qsv`](https://github.com/jqnatividad/qsv) package at version 0.108.0.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/qsv/default.nix [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  buildFeatures = [
+    "feature_capable"
+    "apply"
+    "fetch"
+    "foreach"
+    "generate"
+    "polars"
+  ];
+
+  cargoSha256 = "sha256-8JDlQyHzP/xGNEIHAveyJ7DhA9t7h3O9F+libOT4f+A=";
```
Comment (SuperSandro2000) [inline pkgs/tools/text/qsv/default.nix:28]:
```suggestion
  cargoHash = "sha256-8JDlQyHzP/xGNEIHAveyJ7DhA9t7h3O9F+libOT4f+A=";
```

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by cpcloud** [comment]
@SuperSandro2000 I think I addressed all of your feedback!

**Comment by cpcloud** [comment]
@SuperSandro2000 It looks like the `x86_64-darwin` tests are failing because of a vendored cargo dependencies failure. Is it expected that those dependencies will have different hashes for different platforms?

**Comment by SuperSandro2000** [comment]
> Is it expected that those dependencies will have different hashes for different platforms?

not really. Can you diff them? Maybe we can just rm some files to match them.

**Comment by cpcloud** [comment]
>  Can you diff them?

Not sure how to do that? Do I need a mac? I don't have one at the moment.

**Comment by mimame** [comment]
Hi all, I'm very interested to use qsv on NixOS
I would like to help with the Linux build if needed
Thanks so much in advance and happy new year!

**Comment by uncenter** [comment]
Superseded by #296424.

**Pull request closed without merging**